### DISCUSSION TITLE
Save screenshots to a openGOAL folder in the users Pictures directory

### DISCUSSION THREAD

**Discussion by Zedb0T** [open-goal/jak-project#3276] [General]
rn, Idk where they save on launcher version but I would assume the active directory, and maybe they get deleted on update? idk.

**Comment by ManDude** [comment]
They get saved to `%appdata%` like the savefiles.

**Comment by xTVaser** [comment]
I don't think that's correct, I think Zed is more correct.
- https://github.com/open-goal/jak-project/blob/349919f7880de1d2274379adb98ab3608b54c3d1/common/util/FileUtil.cpp#L692
- uses `file_util::get_file_path` which uses the project path as a base dir https://github.com/open-goal/jak-project/blob/349919f7880de1d2274379adb98ab3608b54c3d1/common/util/FileUtil.cpp#L198-L213

No reference to AppData, and locally testing it this appears to be the case.
![image](https://github.com/open-goal/jak-project/assets/13153231/f879c357-16c1-4d9b-831c-b8fadf5ac678)


If a change is made to this, the directory should imo be configurable as well (could use `tinyfd_selectFolderDialog`). (Also looks like the imgui settings file should also be moved to somewhere more centralized)

**Comment by ManDude** [comment]
You're right, I thought I had made the change a while back. If they're being deleted when the launcher updates the game, then that's pretty bad.
